src/components/VisualPickerOption/index.js

```diff
@@ -1,55 +1,57 @@
+/* eslint-disable react/prop-types */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import { isArray, isString } from 'util';
+import { Consumer as VisualPickerConsumer } from '../VisualPicker/context';
 import { uniqueId } from '../../libs/utils';
 import RenderIf from '../RenderIf';
 import CheckmarkIcon from './checkmark';
 import './styles.css';
 
-/**
- * A VisualPickerOption.
- * @category Form
- */
-export default class VisualPickerOption extends Component {
+class PickerOption extends Component {
     constructor(props) {
         super(props);
         this.inputId = uniqueId('radio');
+        this.handleChange = this.handleChange.bind(this);
     }
 
     getContainerClassNames() {
         const { className } = this.props;
         return classnames('rainbow-visual-picker-option_content-container', className);
     }
 
-    // this need to be discussed with reinier, so not touch it for now.
-    // getType() {
-    //     const { multiple } = this.props;
-    //     if (multiple) {
-    //         return 'checkbox';
-    //     }
-    //     return 'radio';
-    // }
+    getType() {
+        const { multiple } = this.props;
+        return multiple ? 'checkbox' : 'radio';
+    }
 
     isChecked() {
-        // here goes logic to determine if it is checked
-        return false;
+        const { multiple, name, value } = this.props;
+        return multiple
+            ? isArray(value) && value.includes(name)
+            : isString(value) && value !== '' && name === value;
```

Comment:
same use if instead, this is more difficult to understand
in the case of not multiple the ```value !== ''``` it is not necessary because when make the comparation: ```name === value``` it will fail.